digdag-client/src/main/java/io/digdag/client/api/RestWorkflowDefinition.java

```diff
@@ -19,6 +21,9 @@
 
     String getRevision();
 
+    @JsonProperty("timezone")
+    ZoneId getTimeZone();
```

Comment:
I took the same code from https://github.com/treasure-data/digdag/blob/082f72b3b1adcbad570950544b60fc360835dfd3/digdag-client/src/main/java/io/digdag/client/api/RestWorkflowSessionTime.java#L19-L20